content/en/data_security/real_user_monitoring.md

```diff
@@ -0,0 +1,125 @@
+
+You can also transmit all RUM events through your own (proxy) server so that end user devices never directly communicate with Datadog.
+
+#### IP address
+When setting up a RUM application, you can choose whether or not you want to include IP or geolocation data:
+
+{{< img src="data_security/data-security-rum-privacy-compliance-edit-rum-application.png" alt="You can include or exclude geolocation and client IP data from the RUM application setup page" style="width:100%;" >}}
+
+If you choose to collect this data, you can request that it be scrubbed from any future collected data by submitting a support ticket at any time. It is not possible to remove IPs from data collected prior to initiating scrubbing. IP scrubbing occurs at the organization level; this data cannot be removed for specific applications, services, and so on. It is performed on the backend, which means the Browser SDK will still be sending data, but IP addresses will be scrubbed by Datadog backend pipelines and dropped at processing time.
+
+#### Geolocation
+In addition to removing client IPs, you can also choose to scrub geolocation (country, city, county), or GeoIP from all future collected data. It is not possible to scrub only GeoIP data without removing client IPs. As with IP addresses, geolocation scrubbing occurs at the organization level; this data cannot be removed for specific applications, services, and so on. It is done at the backend level, which means the Browser SDK will still be sending data, but geolocation data will be scrubbed out by our backend pipelines and dropped at processing time.
+
+### Proactively search for sensitive data with Sensitive Data Scanner
+[Sensitive Data Scanner][11] allows you to proactively search and scrub sensitive data upon ingestion by Datadog. RUM events are scanned on the stream before any data is stored within Datadog. The tool has the power to scrub, hash, or partially redact PII data before it is stored. It works by applying out-of-the-box or customer-developed pattern matching rules.
+
+## Session Replay-specific privacy options
+See [privacy options specific to Session Replay][12].
+
+### Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /real_user_monitoring/guide/enrich-and-control-rum-data/?tab=event#event-and-context-structure
+[2]: /real_user_monitoring/ios/advanced_configuration/?tab=swift#modify-or-drop-rum-events
+[3]: /real_user_monitoring/android/advanced_configuration/?tab=kotlin#modify-or-drop-rum-events
+[4]: /real_user_monitoring/flutter/advanced_configuration/#modify-or-drop-rum-events
+[5]: /real_user_monitoring/browser/tracking_user_actions/#declare-a-name-for-click-actions
+[6]: /real_user_monitoring/browser/modifying_data_and_context/?tab=npm#user-session
+[7]: /real_user_monitoring/browser/modifying_data_and_context/?tab=npm#user-session
+[8]: /help/
+[9]: /help/
+[10]: /account_management/rbac/permissions/#real-user-monitoring
+[11]: /product/sensitive-data-scanner/
```

Comment:
This link isn't working for me. I'm not sure if this is the article you meant to link to, but this is the URL when I search for Sensitive Data Scanner in the docs: /account_management/org_settings/sensitive_data_detection